app/src/main/java/dev/msfjarvis/aps/data/password/FieldItem.kt

```diff
@@ -14,14 +18,18 @@ class FieldItem(val key: String, val value: String, val action: ActionType) {
   enum class ItemType(val type: String) {
     USERNAME("Username"),
     PASSWORD("Password"),
-    OTP("OTP")
+    OTP("OTP (expires in %ds)"),
   }
```

Comment:
Instead of doing this, can we instead add a new variable called `label` and keep the type as it is? I think that would be a nice separation b/w the `ItemType` and the text that we want to display `label`. However, on second thought, that does feel like extra work for just one field. What do you think?